spec/requests/case_logs_controller_spec.rb

```diff
@@ -209,6 +209,45 @@
         it "shows the download csv link" do
           expect(page).to have_link("Download (CSV)", href: "/logs.csv")
         end
+
+        context "when filtering" do
+          let!(:in_progress_case_log) do
+            FactoryBot.create(:case_log, :in_progress,
+                              owning_organisation: organisation,
+                              managing_organisation: organisation)
+          end
+          let!(:completed_case_log) do
+            FactoryBot.create(:case_log, :completed,
+                              owning_organisation: organisation,
+                              managing_organisation: organisation)
+          end
+
+          it "only shows case logs for selected status" do
+            in_progress_case_row_log = "<a class=\"govuk-link\" href=\"/logs/#{in_progress_case_log.id}\">#{in_progress_case_log.id}</a>"
```

Comment:
I know this is proliferating a style I started but in general it's probably better to use the Capybara methods where possible (`has_selector?`, `has_css?`, `has_link?`, `has_xpath?` etc) than matching escaped html strings like this. 

(Not a blocker for this PR)

Reply:
Hm, would we want to do that for request specs as well as feature?

Reply:
Ideally probably where possible. Not super important to change existing ones but I think it is generally more readable and less brittle?

Reply:
You can just add `let(:page) { Capybara::Node::Simple.new(response.body) }`, you don't need to actually run the test with the Capybara drivers (the slow bit).